### DISCUSSION TITLE
Error Occurred without any message

### DISCUSSION THREAD

**Discussion by Legend94rz** [PaddlePaddle/PaddleSpatial#139] [BaiduKDDCup2022] [answered]
as is shown in the screenshot:
![image](https://user-images.githubusercontent.com/7570644/171612174-0d4cfa9d-2fe4-4356-a7a6-609182bdb2e4.png)

Each submit runs about 1 hour and then fail.

could you please check the server log?

**Comment by ramber1836** [comment]
Please provide your team name.

**Comment by ramber1836** [comment, marked as answer]
This is the log: 
![image](https://user-images.githubusercontent.com/19561583/171640548-15e69c3a-a225-417a-8a7f-227a4158f51d.png)
